src/main/java/domain/LadderResults.java

```diff
@@ -0,0 +1,50 @@
+package domain;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class LadderResults {
+
+    private final HashMap<Integer, LadderResult> ladderResults;
+
+    public LadderResults(final List<String> results, final int numberOfPlayer) {
+        validate(results, numberOfPlayer);
+        this.ladderResults = makeResults(results);
+    }
```

Comment:
정적팩토리 메서드를 만들어 검증을 위한 인자는 팩토리메서드에서 받고 생성자에서는 객체 생성에 필요한 필드만 받아서 구분해줘도 좋을 것 같네요 :) 

```suggestion
	private LadderResults(HashMap<Integer, LadderResult> ladderResults) {
		this.ladderResults = ladderResults;
	}

	public LadderResults of(final List<String> results, final int numberOfPlayer) {
		validate(results, numberOfPlayer);
		return new LadderResults(makeResults(results));
	}
```

Reply:
정적 팩터리 메서드로 해결할 수 있었군요!! 감사합니다 :)